packages/node/package.json

```diff
@@ -13,29 +13,30 @@
     "access": "public"
   },
   "files": [
-    "dist",
-    "types"
+    "dist"
   ],
   "scripts": {
     "clean": "rm -fr dist && mkdir dist",
-    "build": "npm run clean && npm run build:dist",
+    "bundle-types": "../../bin/bundle-types",
+    "build": "npm run clean && npm run build:dist && npm run bundle-types",
     "build:dist": "../../bin/bundle src/notifier.js --node --exclude=iserror,stack-generator,error-stack-parser,request,pump,byline --standalone=bugsnag | ../../bin/extract-source-map dist/bugsnag.js",
-    "test": "bundle exec maze-runner"
+    "test": "bundle exec maze-runner",
+    "postversion": "npm run build"
```

Comment:
The node assets were not built when each version was created. This was the cause.